lib/kube/kubeconfig/kubeconfig_test.go

```diff
@@ -215,6 +216,64 @@ func (s *KubeconfigSuite) TestUpdate(c *check.C) {
 	c.Assert(config, check.DeepEquals, wantConfig)
 }
 
+func (s *KubeconfigSuite) TestUpdateWithExec(c *check.C) {
```

Comment:
Unless the test suite is huge, I generally try to migrate from `check` to standard Go tests when I encounter them as defined in RFD #0001.

This file looks to be only 300-400 lines - care to take a stab at converting it?